update.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
 set -e
 
-cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+if [[  $(uname) == "Darwin" ]]; then
```

Comment:
This might be better written with single brackets since no pattern matching is being done:
```bash
if [  "$(uname)" = 'Darwin' ]; then
```